Added a warning note for small segments sizes (#230)
eProsima/Fast-RTPS-docs: docs/fastdds/transport/shared_memory/shared_memory.rst

   The *kind* value for a SharedMemTransportDescriptor is given by the value
   ``eprosima::fastrtps::rtps::LOCATOR_KIND_SHM``

.. warning::

>     Setting a ``<segment_size>`` close to or smaller than the data size creates a high risk of data loss, as the

Reviewer: ```suggestion
    Setting a ``<segment_size>`` close to or smaller than the data size poses a high risk of data loss, since the
```

>     Setting a ``<segment_size>`` close to or smaller than the size or your data creates a high risk of data loss, as the

Reviewer: Use impersonal language.
